x-pack/legacy/plugins/maps/public/layers/sources/client_file_source/geojson_file_source.js

```diff
@@ -107,8 +107,14 @@ export class GeojsonFileSource extends AbstractVectorSource {
   }
 
   async getGeoJsonWithMeta() {
+    const { features, type } = this._descriptor.featureCollection;
+    const featuresWithoutProps = features.map(({ type, geometry }) =>
```

Comment:
Preserve the properties by copying them over. Otherwise, we'll lose the tooltips.